environment/src/main/java/com/sequenceiq/environment/environment/validation/EnvironmentFlowValidatorService.java

```diff
@@ -99,4 +104,12 @@ public ValidationResult validateAuthentication(EnvironmentValidationDto environm
         return environmentAuthenticationValidator.validate(environmentValidationDto);
     }
 
+    public ValidationResult validateAwsKeysPresent(EnvironmentValidationDto environmentValidationDto, ParametersDto parametersDto) {
+        ValidationResult.ValidationResultBuilder validationResultBuilder = ValidationResult.builder();
+        if (environmentValidationDto.getEnvironmentDto().getCloudPlatform().equals("AWS")) {
```

Comment:
Why a simple string uses a constant is minimum I think, but using the com.sequenceiq.cloudbreak.common.mappable.CloudPlatform would take much more sense and be safer.